packages/fluentui/docs/src/prototypes/table/InteractiveTable.tsx

```diff
@@ -10,6 +19,10 @@ const InteractiveTable = () => {
     setPopupOpen(false);
   };
 
+  const contextMenuItems = ['Add to selection', 'Remove', 'Download'];
+
+  const getRowWithContextMenu = tableRow => <MenuButton contextMenu trigger={tableRow} menu={contextMenuItems} />;
```

Comment:
I would rename this to just `withContextMenu`

Reply:
done :)